eventing/sources/Sources.md

```diff
@@ -0,0 +1,42 @@
+# Knative Event Sources
```

Comment:
Should this file also have the comment that this file should not be manually edited and one should edit the YAML instead? You know for people that don't always remember to read readmes (like me :) )

Reply:
+1

Reply:
Probably. I was trying to do it via a comment in the markdown, but can't find a good way to do so. Should I just write it so that it is visible in the rendered page as well?

Reply:
Done.